File: ValidarEmail.py
import re

def validar_email(email):
    regex = r'^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+(\.[a-zA-Z]{2,})+$'
    return "Email valido" if re.fullmatch(regex, email) else "Email invalido"

print(validar_email("meuemail123@mymail.com"))

File: ValidarEmail.js
const ValidarEmail = (email) => {
    const REGEX = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+(\.[a-zA-Z]{2,})+$/;
    return REGEX.test(email) ? "Email valido" : "Email invalido"
}

console.log(ValidarEmail("meuemail123@mymail.com"));